src/image-tags.js

```diff
@@ -78,6 +81,7 @@ export function ImageTags(seoConfig, config, pageType, data, {url = {}}) {
 
   if(seoConfig.enableTwitterCards) {
     tags.push({name: "twitter:image", content: `https://${config['cdn-image']}/${image.path([16, 9], {w: 1200, auto: "format,compress", ogImage: true})}`})
+    tags.push({property: "twitter:image:alt", content: alt})
```

Comment:
could you please make sure that we have `alt` text . `alt && tags.push({property: "twitter:image:alt", content: alt}) `